packages/network-enablement-controller/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@metamask/network-enablement-controller",
+  "version": "0.0.0",
+  "description": "Provides an interface to the currently network enabled via a MetaMask-compatible provider object",
```

Nit: Same as above. What do you think about rephrasing this to ?
```suggestion
  "description": "Provides an interface to the currently enabled network using a MetaMask-compatible provider object"